src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -128,6 +129,12 @@ export class TxSender {
       }
     }
 
+    // If 1/? threshold and owner chooses to execute created tx immediately
+    // we retrieve txId of newly created tx from proposal response
+    if (isFinalization && txDetails) {
```

Comment:
Wouldn't `if (txDetails)` suffice?

Reply:
Off-chain signatures also propose. It's only when a transaction is created and executed, aka 'finalised', that we should pend based on `txDetails.txId`.

Reply:
(although they should theoretically be the same, they will already be pending)